README.md

```diff
@@ -4,7 +4,7 @@ This is an Akka HTTP hello-world example compatible with Heroku.
 
 ## Usage
 
-To run locally (at `0.0.0.0:8080/v1/hello`):
+To run locally (at `0.0.0.0:8080/v1.0.0/hello`):
```

Comment:
As you are adding API versioning schema, I think it'd be good to include in here some information 'bout it. I mean, generals of the versioning schema.

Reply:
Good idea, I will specify the project and API version in the top of the README as well as the schema they follow.